### DISCUSSION TITLE
[Resolvido] Saldo do Cartão

### DISCUSSION THREAD

**Discussion by anhaabaete** [andreroggeri/pynubank#384] [General]
Há algum método ou função para saber o saldo do cartão de crédito. Ou seja ainda o quanto se tem de crédito?

**Comment by lucasrcezimbra** [comment]
> Há algum método ou função para saber o saldo do cartão de crédito. Ou seja ainda o quanto se tem de crédito?

```python
data = nubank.get_credit_card_balance()

prepaid = Decimal(str(data["prepaid"] / 100))
due = Decimal(str(data["due"] / 100))
future = Decimal(str(data["future"] / 100))

prepaid - due - future
```

**Comment by anhaabaete** [comment]
Funcionou.... 
Mas um só deles já é o saldo.
Obrigado!!!!
